src/sidebar/components/Annotation/AnnotationShareInfo.js

```diff
@@ -1,15 +1,14 @@
 import { Icon, Link } from '@hypothesis/frontend-shared';
-
-import { useStoreProxy } from '../../store/use-store';
-import { isPrivate } from '../../helpers/permissions';
+import classnames from 'classnames';
 
 /**
- * @typedef {import("../../../types/api").Annotation} Annotation
+ * @typedef {import("../../../types/api").Group} Group
  */
 
 /**
  * @typedef AnnotationShareInfoProps
- * @prop {Annotation} annotation
```

Comment:
👍 - Decoupling these components from the store, and thus the app, could prove useful for re-using annotation cards in other apps in future.